driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -389,7 +389,6 @@
       <value compset="_MPASO">48</value>
       <value compset="_DLND.*_MALI">1</value>
       <value compset="_SLND.*SOCN.*_MALI">1</value>
-      <value compset="_DATM.*_SLND.*MPASO.*_MALI">24</value>
```

Comment:
@jonbob, I'm seeing 2 other lines like this that we may also want to fix:
https://github.com/E3SM-Project/E3SM/blob/14847721974bbbea87c20d7e146459c5e9b68798/driver-mct/cime_config/config_component_e3sm.xml#L514
https://github.com/E3SM-Project/E3SM/blob/14847721974bbbea87c20d7e146459c5e9b68798/driver-mct/cime_config/config_component_e3sm.xml#L590

Similar changes will also be needed (sooner or later) for the MOAB coupler.

Reply:
Good point @xylar -- I'll get rid of those as well